pkg/kubelet/rkt/rkt.go

```diff
@@ -754,9 +754,12 @@ func podFinishedMarkerPath(podDir string, rktUID string) string {
 	return filepath.Join(podDir, "finished-"+rktUID)
 }
 
-func podFinishedMarkCommand(touchPath, podDir, rktUID string) string {
-	// TODO, if the path has a `'` character in it, this breaks.
-	return touchPath + " " + podFinishedMarkerPath(podDir, rktUID)
+func podFinishedMarkCommand(touchPath, podDir, rktUID string) (string, error) {
+	apostropheFlag := strings.ContainsRune(touchPath, 39)
```

Comment:
Use the character, not the code, to make this readable (although it's not that much better since it needs to be escaped...)
```
apostropheFlag := strings.ContainsRune(touchPath, '\'')
```